docs: add punctuation & grammar (#659)
nextcord/nextcord: docs/discord.rst

To work with the library and the Discord API in general, we must first create a Discord Bot account.

> Creating a Bot account is a pretty straight forward process.

Reviewer: ```suggestion
Creating a Bot account is a pretty straightforward process.
```

    .. image:: /images/discord_create_app_form.png
        :alt: The new application form filled in.

5. Create a Bot User account by navigating to the "Bot" tab and clicking "Add Bot".

    - Click "Yes, do it!" to continue.

    .. image:: /images/discord_create_bot_user.png
        :alt: The Add Bot button.
6. Make sure that **Public Bot** is ticked if you want other users to invite your bot.

    - You should also make sure that **Require OAuth2 Code Grant** is unchecked unless you
      are developing a service that needs it. If you're unsure, then **leave it unchecked**.

    .. image:: /images/discord_bot_user_options.png
        :alt: How the Bot User options should look like for most people.

7. Get a generated token using the "Reset Token" button. You will have to confirm and enter a 2FA Code. 2FA is a requirement for making Bot Accounts.

>     - **This is not the Client Secret on the General Information page.**

Reviewer 2: ```suggestion
    - **This is not the Client Secret from the General Information page.**
```
this might make more sense

    .. image:: /images/discord_create_app_form.png
        :alt: The new application form filled in.

> 5. Create a Bot User Account by navigating to the "Bot" tab and clicking "Add Bot".

Reviewer: ```suggestion
5. Create a Bot User account by navigating to the "Bot" tab and clicking "Add Bot".
```

Author: It's kind of like saying just "User Account", so I thought adding the capital letter would be better.

Reviewer: I understand it as an account for a Bot User, where account is just like a normal account and doesn't need to be capitalized.

Author: Ah, in that case then, I'll change it back.

    - Click "Yes, do it!" to continue.

    .. image:: /images/discord_create_bot_user.png
        :alt: The Add Bot button.
6. Make sure that **Public Bot** is ticked if you want other users to invite your bot.

>     - You should also make sure that **Require OAuth2 Code Grant** is unchecked, unless you

Reviewer: ```suggestion
    - You should also make sure that **Require OAuth2 Code Grant** is unchecked unless you
```

      are developing a service that needs it. If you're unsure, then **leave it unchecked**.

    .. image:: /images/discord_bot_user_options.png
        :alt: How the Bot User options should look like for most people.

7. Get a generated token using the "Reset Token" button. You will have to confirm and enter a 2FA Code. 2FA is a requirement for making Bot Accounts.

    - **This is not the Client Secret on the General Information page.**

    .. warning::

>         It should be worth nothing that your bot's token is essentially your bot's

Reviewer 2: nothing? noting works here

Author: My bad had thought it was a misspelling

        password. You should **never** share this with anyone else. In doing so,

>         someone can log into your bot and do malicious things, such as leaving

Reviewer: ```suggestion
        someone can log in to your bot and do malicious things, such as leaving
```

        servers, banning all members inside a server, or mentioning everyone maliciously.

        The possibilities are endless, so **do not share this token.**

        If you have accidentally leaked your token, click the "Reset Token" button as soon
        as possible. This revokes your old token and re-generates a new one.
        Now you need to use the new token to log in.

8. After that, click the "Copy" button to copy your token.

    .. image:: /images/discord_bot_copy_token.png
        :alt: The revealed token and the new "Copy" button.

> And that's it. You now have a bot account and you can log onto with that token.

Reviewer 2: im sure in works here too

So you've made a Bot User but it's not actually in any server.

If you want to invite your bot, you must create an invite URL for it.

> 1. Make sure you're logged onto the `Discord website <https://discord.com>`_.

Reviewer: ```suggestion
1. Make sure you're logged in to the `Discord website <https://discord.com>`_.
```

2. Navigate to the `Application page <https://discord.com/developers/applications>`_
3. Click on your bot's page.
4. Go to the "OAuth2" tab.

    .. image:: /images/discord_oauth2.png
        :alt: How the OAuth2 page should look like.

> 5. Tick the "bot" checkbox under "scopes". If you make use of Application Commands (Slash, Message and User Commands) then you should also tick the "applications.commands" scope.

Reviewer: ```suggestion
5. Tick the "bot" checkbox under "scopes". If you make use of Application Commands (Slash, Message and User Commands), also tick the "applications.commands" scope.
```

    .. image:: /images/discord_oauth2_perms.png
        :alt: The permission checkboxes with some permissions checked.

> 7. Now the resulting URL can be used to add your bot to a server. Copy and paste the URL into your browser or Discord client, choose a server to invite the bot to, and click "Authorize".

Author: Fixed 👍🏼

.. note::

>     The person adding the bot requires the "Manage Server" permission to do so.

Reviewer: ```suggestion
    The person adding the bot needs the "Manage Server" permission to do so.
```
